XRTK.ProjectName/Packages/com.xrtk.projectname/Runtime/Providers/Controllers/ProjectNameControllerDataProvider.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Attributes;
+using XRTK.Definitions.Platforms;
+using XRTK.Interfaces.InputSystem;
+using XRTK.ProjectName.Profiles;
+using XRTK.Providers.Controllers;
+
+namespace XRTK.ProjectName.Providers.Controllers
+{
+    [RuntimePlatform(typeof(ProjectNamePlatform))]
+    [System.Runtime.InteropServices.Guid("#INSERT_GUID_HERE#")]
```

Comment:
Can a powershell script generate GUIDs maybe?

Reply:
It already does that